website/pages/docs/plugins/destinations/s3/_authentication.mdx

```diff
@@ -79,3 +79,13 @@ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
 export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
 export AWS_SESSION_TOKEN=<YOUR_SESSION_TOKEN>
 ```
+
+### Using a Custom S3 Endpoint
+
+If you are using a custom S3 endpoint, you can specify it using the `AWS_S3_ENDPOINT` environment variable:
+
+```bash copy
+export AWS_S3_ENDPOINT=https://my-custom-s3-endpoint.com
```

Comment:
I don't think this should be an environment variable... We support AWS environment variables because that is defined by AWS... For this I think we should put this in the spec file. Thoughts?

Reply:
Yes, that might be better. Users could then still use `endpoint: ${AWS_S3_ENDPOINT}` if they wanted to

Reply:
Sure, `AWS_S3_ENDPOINT` looks to be a historical 3rd party add-on anyway.